static/app/views/performance/browser/resources/resourceSummaryPage/sampleImages.tsx

```diff
@@ -128,6 +132,22 @@ function ImageContainer(props: {
   );
 }
 
+function useGeneralProjectSettings(projectId: number) {
```

Comment:
m: Shall we extract this out into a util? And write some tests for it?

Reply:
Yeah i think we should, we we're looking at adding images in the span waterfall view as well, so this might be used in the future. Will update!

Reply:
I made the change, but i didn't write any tests. We're you thinking of testing the hook specifically, or the usage of the hook in the resource module.